### PR TITLE
Fix importMCISet and rerun to update data

### LINKED ISSUE #351: Apostrophes in Rulings are not normalised.
Introduced in a7c605cf4a3540aba45040b174d84fd0809be1f0 (#328), rulings now use left single quotes for apostrophes instead of the normalised `'`.

### PR DESCRIPTION
**Pull request by gwax**
This PR fixes:

-   crash in importMCISet on undefined `innerHTML` (should be `textContent`)
-   smart quotes not being corrected to regular quotes in mci imports
-   smart quotes not being corrected to regular quotes in rulings
-   fixes #351 
-   CED Will-o'-the-wisp not matching printings in other sets
-   color identity in mci imports
-   fix a missing artist on a card
-   rerun all of the mci imports

### CODE REVIEW

File: shared/C.js [hunk 1]
```diff
@@ -3756,7 +3756,8 @@ Array.prototype.pushAll = function(otherArray) {
 			{ match : {name : "Howlpack Alpha"}, replace : {layout : "double-faced", names : ["Mayor of Avabruck", "Howlpack Alpha"]}},
 			{ match : {name : "Overbeing of Myth"}, replace : {flavor : "\"She walks among us unseen, learning from our imperfections.\"\n—The Seer's Parables"}},
 			{ match : {name : "Wren's Run Packmaster"}, replace : {releaseDate : "2007-09-29"}},
-			{ match : {name : "Sultai Ascendacy"}, replace : {name : "Sultai Ascendancy", imageName : "sultai ascendancy"}}
+			{ match : {name : "Sultai Ascendacy"}, replace : {name : "Sultai Ascendancy", imageName : "sultai ascendancy"}},
+			{ match : {name : "Nissa, Vastwood Seer"}, replace : {artist : "Wesley Burt"}}
```
Comment (fenhl) [inline shared/C.js]: Why is this line added?

File: json/CED.json [hunk 2]
```diff
@@ -15696,7 +15944,6 @@
         "BTD",
         "INV",
         "7ED",
-        "ODY",
```
Comment (fenhl) [inline json/CED.json]: Odyssey printing is removed from Mountain. That seems wrong.

File: json/pCEL.json [hunk 3]
```diff
@@ -70,7 +80,7 @@
       "subtypes": [
         "Dragon"
       ],
-      "text": "When Shichifukujin Dragon comes into play, put seven +1/+1 counters on it.\n {R}{R}{R}, Remove two +1/+1 counters: Put three +1/+1 counters on Shichifukujin Dragon at end of turn. Play this ability as a sorcery.",
+      "text": "When Shichifukujin Dragon comes into play, put seven +1/+1 counters on it.\n\n {R}{R}{R}, Remove two +1/+1 counters: Put three +1/+1 counters on Shichifukujin Dragon at end of turn. Play this ability as a sorcery.",
```
Comment (fenhl) [inline json/pCEL.json]: Shichifukujin Dragon's text has two consecutive newlines now.

File: json/pGTW.json [hunk 4]
```diff
@@ -2004,15 +2036,15 @@
       "rulings": [
         {
           "date": "2008-08-01",
-          "text": "Each of the triggered abilities has an \"intervening 'if' clause.\" That means (1) the ability won't trigger at all unless you control two or more lands of the appropriate land type when the Hedge-Mage enters the battlefield, and (2) the ability will do nothing if you don't control two or more lands of the appropriate land type by the time it resolves."
+          "text": "Each of the triggered abilities has an \"intervening ‘if' clause.\" That means (1) the ability won't trigger at all unless you control two or more lands of the appropriate land type when the Hedge-Mage enters the battlefield, and (2) the ability will do nothing if you don't control two or more lands of the appropriate land type by the time it resolves."
```
Comment (fenhl) [inline json/pGTW.json]: Here, a `'` is changed into a `‘`. This seems like the opposite of the intended change.

File: shared/C.js (hunk 1 above)
Comment (gwax) [inline shared/C.js]: MagicCards.info doesn't have the artist listed and results in the card having an empty string for artist. This fixes that.

File: json/pGTW.json (hunk 4 above)
Comment (gwax) [inline json/pGTW.json]: Will fix.

File: json/pCEL.json (hunk 3 above)
Comment (gwax) [inline json/pCEL.json]: I'll look into that.

File: json/CED.json (hunk 2 above)
Comment (gwax) [inline json/CED.json]: Weird, I'll look into this.

Comment (gwax) [inline json/CED.json]: This turns out to be a problem with the build on my end. ODY was out of date and this build wrong. Fix will be pushed momentarily.

File: json/pCEL.json (hunk 3 above)
Comment (gwax) [inline json/pCEL.json]: Fixed.

**Comment by fenhl** [comment]
Looks good now as far as I can tell.

**Pull request merged**